### PR TITLE
Create pkgbuilds in archlinux/ folder

### PR DESCRIPTION
**Pull request by Jai-JAP**

### CODE REVIEW

File: archlinux/PKGBUILD-arm64 [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Maintainer: Jai-JAP <parjailu@gmail.com>
+# Contributor: Sebastien Chevalier <sebastien.chev@gmail.com>
```
Comment (ptitSeb) [inline archlinux/PKGBUILD-arm64:2]: Can you put my box86 address instead: ptitseb@box86.org

File: archlinux/PKGBUILD-rk3399 [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Maintainer: Jai-JAP <parjailu@gmail.com>
+# Contributor: Sebastien Chevalier <sebastien.chev@gmail.com>
```
Comment (ptitSeb) [inline archlinux/PKGBUILD-rk3399:2]: Same

File: archlinux/PKGBUILD-rpi4 [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Maintainer: Jai-JAP <parjailu@gmail.com>
+# Contributor: Sebastien Chevalier <sebastien.chev@gmail.com>
```
Comment (ptitSeb) [inline archlinux/PKGBUILD-rpi4:2]: same

File: archlinux/PKGBUILD-tegra [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# Maintainer: Jai-JAP <parjailu@gmail.com>
+# Contributor: Sebastien Chevalier <sebastien.chev@gmail.com>
```
Comment (ptitSeb) [inline archlinux/PKGBUILD-tegra:2]: and same

File: archlinux/PKGBUILD-x86_64 [hunk 5]
```diff
@@ -0,0 +1,44 @@
+# Maintainer: Jai-JAP <parjailu@gmail.com>
+# Contributor: Sebastien Chevalier <sebastien.chev@gmail.com>
```
Comment (ptitSeb) [inline archlinux/PKGBUILD-x86_64:2]: and same again

File: archlinux/PKGBUILD-arm64 (hunk 1 above)
Comment (Jai-JAP) [reply archlinux/PKGBUILD-arm64]: Done ...
Take a look

File: archlinux/PKGBUILD-x86_64 (hunk 5 above)
Comment (Jai-JAP) [reply archlinux/PKGBUILD-x86_64]: done

File: archlinux/PKGBUILD-tegra (hunk 4 above)
Comment (Jai-JAP) [reply archlinux/PKGBUILD-tegra]: done

File: archlinux/PKGBUILD-rpi4 (hunk 3 above)
Comment (Jai-JAP) [reply archlinux/PKGBUILD-rpi4]: done

File: archlinux/PKGBUILD-rk3399 (hunk 2 above)
Comment (Jai-JAP) [reply archlinux/PKGBUILD-rk3399]: done

**Comment by Jai-JAP** [comment]
pl wait a bit before merging

**Comment by ptitSeb** [comment]
> pl wait a bit before merging

Ok, I wait.

**Comment by Jai-JAP** [comment]
I have added your name as a Author instead as a Collaborator

**Comment by Jai-JAP** [comment]
This is now ready to merge

**Pull request merged**